api/services/diff/v1/diff.proto

```diff
@@ -23,10 +23,11 @@ service Diff {
 }
 
 message ApplyRequest {
+	string snapshotter = 1;
```

Comment:
Maintainers plz let me know requiring snapshotter is correct design.
In this design, I assumed the differ instance can be determined by solely specifying snapshotter instance name. 
Alternative design is to require `string differ` explicitly instead of `string snapshotter`.